### DISCUSSION TITLE
Excited!!!

### DISCUSSION THREAD

**Discussion by ckiyme** [nanojsx/nano#144] [General]
Just found out about this last night, and I'm so excited about it.
Is the plan to be a competitor to Preact and Lit?
Anyway, can't wait to see it ready for production. :)

**Comment by yandeu** [comment]
I do not plan to compete.

I would say it is already production ready since I use it for most of my projects for months without any issues. 
But I can only say that about the features I personally use. For example, I have never used `<Suspense />`, the CustomElementsMode or the Context API.

Personally I use it a lot for client-side only projects. Just using a tsconfig file and esbuild. Fast and easy to setup.

Also, since the core is stable and nanojsx has all the features I need, I don't expect many changes in the future and can therefore focus on stability.
This is my focus, having a great and stable library for many future projects. Lightweight, huge flexibility and therefore very powerful. 😊

**Reply by ckiyme** [reply]
Thank you very much for your answer!
Do you mind sharing any of the projects that you're building with nano, please? 😄

**Reply by yandeu** [reply]
I have one project that I could opensource maybe next year. But else, all my projects are private 🙁
